doc/data_protocol.md

```diff
@@ -138,43 +197,54 @@ Commands from the Ground Station to the Engine Computer
     <th>Data size</th>
     <th>Data</th>
     <th>Comment</th>
+    <th>Tx Rate</th>
   </tr>
 </thead>
 <tbody>
   <tr>
-    <td>0x00</td>
-    <td>Time sync<br>Can be sent by FC</td>
-    <td>-</td>
-    <td>-</td>
-    <td></td>
+    <td>0x20</td>
+    <td>N/A</td>
+    <td> </td>
+    <td> </td>
+    <td> </td>
+    <td> </td>
   </tr>
   <tr>
-    <td>0x01</td>
-    <td>Set power mode</td>
-    <td>1 Byte</td>
-    <td>TBD enum</td>
-    <td>Default state is TBD</td>
+    <td>0x21</td>
+    <td>N/A</td>
+    <td> </td>
+    <td> </td>
+    <td> </td>
+    <td> </td>
   </tr>
   <tr>
-    <td>0x02</td>
-    <td>Set engine state</td>
-    <td>1 Byte</td>
-    <td>bit0: is_launch_aborted<br>bit1: is_engine_armed<br>bit2: is_engine_en</td>
-    <td>Default state is 0<br>bit0 has absolute priority<br>Must set is_engine_armed to 1 <br>before engine can be enabled</td>
+    <td>0x22</td>
+    <td>Set Power Mode</td>
+    <td>1</td>
+    <td>TBD</td>
+    <td></td>
+    <td> </td>
   </tr>
   <tr>
     <td>0x03</td>
-    <td>Set liftoff status</td>
-    <td>1 Byte</td>
-    <td>bit0: is_liftoff_confirmed</td>
-    <td>Confirmed by FC sensors</td>
+    <td>Set Engine State</td>
+    <td>1</td>
+    <td>bit0: is_launch_aborted<br>bit1: is_engine_armed<br>bit2: is_engine_en </td>
+    <td>Default state is 0, bit 0 has<br>absolute priority. Must set<br>is_engine_armed to 1 before<br>engine can be enabled </td>
+    <td> </td>
+  </tr>
+  <tr>
+    <td>0x2F</td>
+    <td>Fire Rocket</td>
+    <td>1</td>
+    <td>TBD</td>
+    <td>Will send the signal to launch<br>the rocket</td>
+    <td></td>
   </tr>
 </tbody>
 </table>
 
->**Note:** The LoRa link adds a sizeable delay in the transmission of the commands. This delay makes the "Time sync" frame (`0x00`) inaccurate if sent from the ground. This frame is intended to be sent from the Ground Station for testing purposes and in the absence of GNSS receiver connected to the Flight Controller. When a GNSS receiver is connected to the Flight Controller, the Flight Controller sends a "Time sync" frame based on GNSS time to the Engine Computer over the CAN Bus
-
-Returned data
+Replies sent from the Engine Computer to the Ground Station
```

Comment:
I'd suggest removing this table for the time being as I expect that this will change rapidly and often in the coming weeks as development ramps up, but adding the following:

```suggestion
Replies sent from the Engine Computer to the Ground Station:

- Time sync requests
- Hardware / software statuses (power mode & errors)
- ... Potentially more
```

Reply:
As you can see almost all the table is TBD, is most about having and idea of what we should expect to take it into account for the dashboard. We can replace it for some text.

Reply:
A table suggests that the actual binary data will be similar to what is written here, but I can't give any promises to that 😄